src/Symfony/Component/HttpKernel/Profiler/Profile.php

```diff
@@ -69,16 +65,14 @@ public function getToken()
     }
 
     /**
-     * Sets the parent token.
+     * @param self
```

Comment:
How about adding the parameter name here, too?

Reply:
This docblock is actually not need as the parameter has a type hint.